src/views/collection-view/components/collection-tutorial-list/index.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames'
+import TutorialCard from 'components/tutorial-card'
+import { TutorialCardPropsWithId } from 'components/tutorial-card/types'
+import s from './collection-tutorial-list.module.css'
+
+function CollectionTutorialList({
+  tutorials,
+  isOrdered,
+}: {
+  tutorials: TutorialCardPropsWithId[]
+  isOrdered: boolean
+}) {
+  const ListRoot = isOrdered ? 'ol' : 'ul'
```

Comment:
Oh that's interesting, I didn't realize you could define the element just via a string like this! Cool 😎